config/rbac/role.yaml

```diff
@@ -6,32 +6,6 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
-- apiGroups:
```

Comment:
`make manifests` command deletes `role` config related to the volumeReplicationClass